cueimports.go

```diff
@@ -325,6 +325,8 @@ func resolveInPackage(unresolved map[string][]string, resolved map[string]string
 							rel = strings.TrimPrefix(rel, "cue.mod/pkg/")
 							rel = strings.TrimPrefix(rel, "cue.mod/usr/")
 							resolved[pkgName] = rel
+						} else if rel != pkgName {
+							resolved[pkgName] = filepath.Join(modName, rel) + ":" + pkgName
```

Comment:
I have very little experience writing Go code. If it's more conventional to use `fmt.Sprintf` I'm happy to make that change.

Reply:
This kind of concatenation is completely valid :)